### ISSUE TITLE
Broken WiFi provisioning after app switching.

### ISSUE THREAD

**Issue by mvacha** [espressif/esp-idf-provisioning-ios#14]
I'm unable to complete WiFi provisioning if I switch apps on the WiFi password dialog.
I'm using ESP-IDF v4.1-beta2, iOS app 2.0.4
Log:
```
I (23971) protocomm_httpd: Creating new session: 57
W (25401) protocomm_httpd: Closing session as socket 57 was closed
I (25411) protocomm_httpd: Creating new session: 57
E (25461) security1: Security version mismatch. Closing connection
E (25461) protocomm_httpd: Data handler failed
W (25461) httpd_uri: httpd_uri: uri handler execution failed
``

**Comment by vikas-chandra-mnnit** [comment]
@mvacha Thanks for informing us. We are looking into this issue and will update you soon regarding it.

**Comment by shahpiyushv** [comment]
@mvacha , can you please apply [This patch](https://github.com/espressif/esp-idf-provisioning-ios/files/5022504/0001-protocomm_httpd-Restart-security-session-if-request.txt) in esp-idf and see if it resolves the issue? This has already been merged into master branch and will soon be available even on release/v4.1 branch.

**Comment by shahpiyushv** [comment]
Closing this issue since appropriate solution has been provided. Please fell free to reopen/file new issues if required.
